packages/upgrade/transforms/__tests__/enable-v12-overflowmenu-test.js

```diff
@@ -0,0 +1,18 @@
+// __tests__/enable-v12-overflowmenu-test.js
```

Comment:
To match the others in this folder
```suggestion
/**
 * Copyright IBM Corp. 2025
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

'use strict';

```